fetcher.go

```diff
@@ -284,6 +284,11 @@ func updateSuccessServerTimingMetrics(timingHeaders []string, resourceType strin
 					fetchDurationPerPeerSuccessCacheMissTotalLassieMetric.WithLabelValues(resourceType).Observe(float64(m.Duration.Milliseconds()))
 				case "nginx":
 					fetchDurationPerPeerSuccessTotalL1NodeMetric.WithLabelValues(resourceType, cache_status).Observe(float64(m.Duration.Milliseconds()))
+					networkTimeMs := totalTimeMs - m.Duration.Milliseconds()
+					fetchNetworkSpeedPerPeerSuccessMetric.WithLabelValues(resourceType).Observe(float64(recieved) / float64(networkTimeMs))
```

Comment:
what about when `networkTimeMs` is 0? - that leads to a division by 0

Reply:
Fixed and added more sanity checks.